app/controllers/application_controller.rb

```diff
@@ -6,13 +6,13 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   protect_from_forgery with: :exception
 
-  before_action :ensure_user_is_logged_in
+  before_action :ensure_user_is_logged_in, except: [:download_json]
```

Comment:
Other metadata types have a `download_json` method as well, so this would skip in unintended places. Also, I think in general it is better to push down the skipping or excepting of before actions as far down as possible in the controller inheritance chain. So I think you probably should remove the `except`s here and skip the before action in the various collection controllers